src/neural_network/layer/neuron/RecurrentNeuron.cpp

```diff
@@ -37,6 +38,7 @@ std::vector<float>& RecurrentNeuron::backOutput(float error)
 {
     error = error * outputFunction->derivative(this->sum);
 
+    //#pragma omp simd // seems to do nothing
```

Comment:
Replace `#pragma omp simd` with `#pragma omp parallel for num_threads(/*number of logical cores*/) schedule(runtime)`

Reply:
I have to think if I want to set multi-core in this branch, if so I have to think about how to configure the number of cores to use in the library.